docs/wiki/User Guide/Configurations.md

```diff
@@ -155,7 +155,8 @@ The following configurations are inherited from the open source Kafka client con
+| default.replication.throttle                                       | Long	| N         | null	                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | The replication throttle applied to replicas being moved, in bytes per second.	                                                                                                                                                                                                                                                                                             |
+| default.log.dir.throttle                                           | Long	| N         | null	                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | The replication throttle applied to replicas being moved, in bytes per second.	                                                                                                                                                                                                                                                                                             |
```

Comment:
Please add different descriptions for both config since one of them is inter broker and the other is intra broker.

Reply:
OMG. What a silly mistake 😓 fixed.